git/00.setup.md

```diff
@@ -66,37 +66,7 @@ alias gs="git status -sb"
 ## Git Status in Command Prompt
 
 Another useful part of git config is to have the branch names and status of the
-working directory expressed in your command prompt when in a git repo. Below is
-an example you can add to your `.bash_profile` as are starting point.
-
-```bash
-# in .bash_profile
-function _git_color() {
```

Comment:
This approach is still valid if you want to not have a dependency.  Might be worth moving to a gist